src/openseadragon.js

```diff
@@ -508,7 +518,7 @@
   *     Milliseconds to wait after each tile retry if tileRetryMax is set.
   *
   * @property {Boolean} [useCanvas=true]
-  *     Set to false to not use an HTML canvas element for image rendering even if canvas is supported.
+  *     Deprecated. Use drawer option to specify preferred renderer.
```

Comment:
I originally read this as `drawerOptions` and got confused. I don't know if there's a better way to say this to avoid that possible confusion, so maybe it's fine... Just noting it.

Reply:
I tried rewording a bit. Not sure how much it helped if any. I'm open to other suggestions too.